pkg/logger/package_logger.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logger
+
+import "github.com/coreos/pkg/capnslog"
+
+// NewPackageLogger wraps "*capnslog.PackageLogger" that implements "Logger" interface.
+//
+// For example:
+//
+//  var defaultLogger Logger
+//  plog := capnslog.NewPackageLogger("github.com/coreos/etcd", "snapshot")
+//  defaultLogger = NewPackageLogger(plog)
+//
+func NewPackageLogger(p *capnslog.PackageLogger) Logger {
```

Comment:
probably do an assert that Logger satisfies grpc.Logger

Reply:
/cc @gyuho

Reply:
something like

```
var _ MyInterface = MyType{}
```

Reply:
Just added.